nibabel/parrec.py

```diff
@@ -244,6 +245,9 @@ class PARRECError(Exception):
     pass
 
 
+# Value after colon may be absent
+GEN_RE = re.compile(r".\s+(.*?)\s*:(.*)")
```

Comment:
Eric - the fix is here - this regex allows the field after the colon to be whitespace only - in the example file the Patient name was empty, which caused the previous code to barf.